package.json

```diff
@@ -18,30 +22,31 @@
     "homepage": "https://github.com/GPII/gpii-express-user",
     "dependencies": {
         "foundation-sites": "6.3.1",
```

Comment:
foundation-sites 6.6.3 is available if suitable

Reply:
Their .1 releases have often required more energy than I can muster.  I will give it a try and see how it looks.

Reply:
Looks fine.